raphtory/Cargo.toml

```diff
@@ -73,6 +73,7 @@ pometry-storage = { workspace = true, optional = true }
 prost = { workspace = true, optional = true }
 prost-types = { workspace = true, optional = true }
 
+
 [target.'cfg(target_os = "macos")'.dependencies]
 snmalloc-rs = { workspace = true }
```

Comment:
are we going with this one? it's the least supported option (perhaps we should create an issue about this)

Reply:
We should probably raise an issue (I had a go at reproducing it with a simple example but couldn't manage). This one is noticeably faster than jemalloc for this particular scenario so I left it for now.

Reply:
Fine, long term I don't know what is the commitment of Microsoft to supporting this allocator